pkg/api/helper/helpers.go

```diff
@@ -113,9 +113,18 @@ func IsStandardContainerResourceName(str string) bool {
 }
 
 // IsOpaqueIntResourceName returns true if the resource name has the opaque
-// integer resource prefix.
+// integer resource prefix, or it is another unknown resource name not in the
+// default namespace.
 func IsOpaqueIntResourceName(name api.ResourceName) bool {
-	return strings.HasPrefix(string(name), api.ResourceOpaqueIntPrefix)
+	return IsCustomNamespacedResourceName(name)
```

Comment:
Reserving `kubernetes.io` domain for first class resources is what I assumed this PR is introducing. If that's true, `alpha.kubernetes.io` is a subdomain of `kubernetes.io` and it should also be considered to be a first class resource. 
Any compute resource under the `kubernetes.io` or the default domain has to be explicitly added to the system across the scheduler, admission, node, quota, limitrange, UI, etc.

For now, all non-first class resources are considered to be integer resources without any QoS semantics (requests == limits).